deployment_report/model/utils/ingress_util.py

```diff
@@ -120,51 +120,57 @@ def get_ingress_version(kubectl: KubectlInterface) -> Optional[Text]:
     """
 
     version: Text = ""
+
     getpod_cmd: AnyStr = "get pods -n " + kubectl.ingress_ns + \
```

Comment:
maybe for future regression safety go ahead and add an early check for kubectl.ingress_ns that can return immediately if not set

```
if not kubectl.ingress_ns:
    return version

getpod_cmd: AnyStr = "get pods -n " + kubectl.ingress_ns ...
```

Reply:
Code updated.